### ISSUE TITLE
Black&White Emojis

### ISSUE THREAD

**Issue by selimgr** [snapcrafters/discord#30]
I get the B&W Emojis only on Discord. I don't understand why, I'm using Noto Color Emoji font and I'm on Ubuntu 18.04 LTS. They usually work on every apps that I have but on Discord I still get the B&W Emojis (they are showed correctly in the title bar).

![Screen capture](https://i.imgur.com/BTxmitO.png)

Is something missing ? or is it related to the app?

**Comment by Fuseteam** [comment]
that has to do with the app itself i believe
i have in all of discord not seen color emoji in the channel names so far

**Comment by selimgr** [comment]
I think so

**Comment by IoIxDButStronger** [comment]
No, I'm here from Google Search and I believe this is a problem with Discord itself, as evident by the fact that I don't even use the snap package and I have this problem. It's also evident by the fact I changed the emoji font to Twemoji Color Font and I actually noticed that the emojis DID change, but they're still black and white outlines. I think Discord simply does not allow for color emojis. There's something hard coded into the app that disallows it.

**Comment by Fuseteam** [comment]
@IoIxDButStronger are you sure you installed you emoji font correctly? i have seen reports of it working correctly in the chris titus tech discord server

**Comment by IoIxDButStronger** [comment]
I ran the install.sh and set the default font to Bitstream Vera, I don't know what else I would have to do to make it work.

**Comment by Fuseteam** [comment]
check www.christustech.com/emoji perhaps you can fix it for the local deb installation

**Comment by Joellidin** [comment]
I have tried modifying the fontconfig file, but it does nothing for me. Anyone that got the colors to work and have some tips?

**Comment by Fuseteam** [comment]
considering the issue is still open, i doubt it. are you using the snap btw? the font config is not in its usual place in that case

**Comment by Joellidin** [comment]
> considering the issue is still open, i doubt it. are you using the snap btw? the font config is not in its usual place in that case

I am using the snap. I did not realize it could be somewhere else. Where can I find the font config when using snap?

**Comment by Fuseteam** [comment]
snap files are usually in `/snap`, `/var/snap` or `~/snap` iirc

**Comment by Joellidin** [comment]
Thank you very much. I got it to work now by changing the correct font config file. Appreciate it.

**Comment by Fuseteam** [comment]
awesome! can you share where the correct font config file is for future reference?

**Comment by Joellidin** [comment]
I changed the `$HOME/snap/discord/122/.config/fontconfig/fonts.conf` file.

**Comment by merlijn-sebrechts** [comment]
Thanks for reporting everyone!!

This appears to be fixed now in the latest version of the snap, so I'm closing this issue now. Please comment if this is not the case.
